### ISSUE TITLE
[BUG]: Power BI template refresh fails on first launch

### ISSUE THREAD

**Issue by pratapladhani** [microsoft/powerapps-tools#221]
**Describe the bug**
Power BI Report template has issues when loading data

**To Reproduce**
1. Open the Power BI template in Power BI Desktop as described on https://docs.microsoft.com/en-us/powerapps/sample-apps/emergency-response/deploy-configure#publish-the-power-bi-dashboard
2. Enter the Organization Name & CDS Base Solution URL
3. Click on "Load"
4. Refresh fails with the following error :

Facilities
The name 'Vents in use' is already used for a measure on table 'Equipment Needed'. Choose a different name.
Locations
The name 'Vents in use' is already used for a measure on table 'Equipment Needed'. Choose a different name.
Supplies
The name 'Vents in use' is already used for a measure on table 'Equipment Needed'. Choose a different name.
Supply Trackings
The name 'Vents in use' is already used for a measure on table 'Equipment Needed'. Choose a different name.
Staffing Needed
The name 'Vents in use' is already used for a measure on table 'Equipment Needed'. Choose a different name.
Staffing Types
The name 'Vents in use' is already used for a measure on table 'Equipment Needed'. Choose a different name.
OrganizationName
The name 'Vents in use' is already used for a measure on table 'Equipment Needed'. Choose a different name.
ServiceRootURL
The name 'Vents in use' is already used for a measure on table 'Equipment Needed'. Choose a different name.
Census
The name 'Vents in use' is already used for a measure on table 'Equipment Needed'. Choose a different name.
COVID
The name 'Vents in use' is already used for a measure on table 'Equipment Needed'. Choose a different name.
Equipment Needed
The name 'Vents in use' is already used for a measure on table 'Equipment Needed'. Choose a different name.
Last Refresh Date
The name 'Vents in use' is already used for a measure on table 'Equipment Needed'. Choose a different name.
Facility Discharge Info
The name 'Vents in use' is already used for a measure on table 'Equipment Needed'. Choose a different name.

**Expected behavior**
Refresh should complete successfully. 

**Screenshots**

**Interface(please complete the following information):**
 - Solution downloaded from https://aka.ms/emergency-response-solution [https://github.com/microsoft/powerapps-tools/blob/master/Apps/EmergencyResponse/EmergencyResponseApp.zip] on 4/12/2020
 - Power BI Desktop version 2.80.5803.421 64-bit (April 2020)

**Additional context**
Steps taken to resolve the issue: Renamed "Vents in use" Measure in Equipment Needed table to "Vents-in-use"

**Comment by dpsbali** [comment]
Where can we download Power BI Desktop version 2.80.5803.421 64-bit (April 2020)

**Comment by dpsbali** [comment]
Yes, that's how I also download the tool but the version that gets installed is Version: 2.79.5768.1082 64-bit (March 2020) and it errors complaining that the queries  were authored  with a newer  version of Power BI Desktop and might not work with your version. Probably the one mentioned above in documentation is not generally available.

**Comment by narpad** [comment]
Can you please download the latest update and let us know if the problem still persists?

**Comment by dpsbali** [comment]
Hi, I checked in the latest update the problem is not there anymore but think there is an upgrade on the entities the latest report is complaining for some missing columns, like msft_location in supply tracking.
